Jenkinsfile.promote

```diff
@@ -152,20 +163,23 @@ boolean isDeployLatestTag(){
     return params.DEPLOY_WITH_LATEST_TAG
 }
 
-void loginOpenshiftRegistry() {
-    // Use creds & route as `jenkins-slave` is not running directly on Openshift
-    withCredentials([string(credentialsId: 'openshift-login-passwd', variable: 'OC_PASSWORD')]) {
-        sh "set +x && oc login -u admin -p ${OC_PASSWORD} --server=${OPENSHIFT_API} --insecure-skip-tls-verify"
-        sh "set +x && docker login -u admin -p \$(oc whoami -t) ${OPENSHIFT_REGISTRY_ROUTE}"
+void loginOpenshift(){
+    withCredentials([usernamePassword(credentialsId: "OPENSHIFT_CREDS", usernameVariable: 'OC_USER', passwordVariable: 'OC_PWD')]){
+        sh "oc login --username=${OC_USER} --password=${OC_PWD} --server=${OPENSHIFT_API} --insecure-skip-tls-verify"
     }
 }
 
+void loginOpenshiftRegistry(){
+    loginOpenshift()
+    sh "set +x && podman login -u jenkins -p \$(oc whoami -t) --tls-verify=false ${OPENSHIFT_REGISTRY}"
```

Comment:
we probably should not expose any user information here, can we use envs instead?

Reply:
+1 we can store this in Jenkins credentials

Reply:
as for images, username is useless but needed.
https://github.com/kiegroup/kogito-images/pull/182#discussion_r447716950